### PR TITLE
As a system, I want to email all students when there is a new assignment assignment in a course

### PR DESCRIPTION
**Pull request by HuynhThanhCanh**
Mail demo:
image

### If you are using node version lower than 12.22.0 please update to version 12.22.0 or higher

Link task: https://app.clickup.com/t/9bzzuj

### CODE REVIEW

**Comment by duongdev** [comment]
Task linked: [CU-9bzzuj As a system, I want to email all students when there is a new assignment assignment in a course](https://app.clickup.com/t/9bzzuj)

File: packages/server/package.json [hunk 1]
```diff
@@ -20,7 +20,8 @@
     "test:e2e": "cross-env NODE_ENV=test jest --config ./test/jest-e2e.json"
   },
   "dependencies": {
-    "@nestjs/common": "7.6.18",
+    "@nestjs-modules/mailer": "^1.6.0",
```
Comment (danghieuliem) [inline packages/server/package.json:23]: ông xoá dấu ^ đi lúc trước anh Dương có nói không được để á, xem chỉnh bên dưới luôn nha

Comment (HuynhThanhCanh) [reply packages/server/package.json:23]: okkay👌

**Review by nhatminh0509 — APPROVED** [review]

**Review by nguyenvanhai-0911 — APPROVED** [review]

**Review by danghieuliem — APPROVED** [review]

**Review by nguyenvanhai-0911 — APPROVED** [review]

**Comment by nhatminh0509** [comment]
sửa lại link ảnh đê

**Review by danghieuliem — APPROVED** [review]

**Review by nhatminh0509 — APPROVED** [review]

**Review by duongdev — APPROVED** [review]

**Pull request merged**